readthedocs/rtd_tests/tests/test_api.py

```diff
@@ -968,6 +969,40 @@ def test_get_version_by_id(self):
             version_data,
         )
 
+    def test_get_active_versions(self):
+        """
+        Test the full response of ``/api/v2/version/?project__slug=pip&active=true``
+        """
+        pip = Project.objects.get(slug='pip')
+
+        data = QueryDict('', mutable=True)
+        data.update({
+            'project__slug': pip.slug,
+            'active': 'true',
+        })
+        url = '{base_url}?{querystring}'.format(
+            base_url=reverse('version-list'),
+            querystring=data.urlencode()
+        )
+
+        resp = self.client.get(url, content_type='application/json')
+        self.assertEqual(resp.status_code, 200)
+        self.assertEqual(resp.data['count'], pip.versions.filter(active=True).count())
+
+        # Do the same thing for inactive versions
```

Comment:
nit: this could be achieved in a fancy way by using pytest parametrize: https://docs.pytest.org/en/latest/parametrize.html

Reply:
I actually think that would make the test less clear.

Reply:
Parametrize doesn't work here anyway https://docs.pytest.org/en/latest/unittest.html#pytest-features-in-unittest-testcase-subclasses